### DISCUSSION TITLE
Widget open app with widget's filter/sort

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [TechbeeAT/jtxBoard#634] [Ideas]
Hello!

I'm playing around with creating multiple widgets: one for "Today (all)", one for "Just Chores", and so on.

I think it would really speed up one's ability to navigate jtxBoard if my curated filter/sort for the tapped widget were automatically applied when opening the app from said widget.

When I tap my "Today (all)" widget's header, back to launcher, then tap "Just Chores" widget header- each should drop me into the view already defined in the widget settings.

I wasn't sure if I should have submitted this over in issues, but I just assume that whenever a developer makes a discussion board they want all feature requests in it to reduce clutter. Let me know if this belongs elsewhere.

Thanks!

**Comment by patrickunterwegs** [comment]
Hey @AeliusSaionji !
Thank you for your suggestion!
Sure it's a nice idea and should be feasible. 
There are two options for that: 
- Open the app at the corresponding view and overwrite the active filter settings
- Open the app in a filter view (for example when you open an entry with a category, then click on the category, you would return to a view with all entries that have that specific category, but without the option to do a further filtering).

I would say the second option would be better and to not overwrite the current filter set in the app. What do you think?

**Reply by AeliusSaionji** [reply]
The second sounds great to me!
